src/bootstrap.coffee

```diff
@@ -40,9 +40,10 @@ loadPreloadedApps = ->
 		fs.readFileAsync(appsPath, 'utf8')
 		.then(JSON.parse)
 		.map (app) ->
-			utils.extendEnvVars(app.env, userConfig.uuid, app.appId, app.name, app.commit)
+			utils.extendEnvVars(app.env, userConfig.uuid, userConfig.deviceApiKey, app.appId, app.name, app.commit)
 			.then (extendedEnv) ->
 				app.env = JSON.stringify(extendedEnv)
+				app.markedForDeletion = false
```

Comment:
What is this about?

Reply:
This is part of the second commit - the commit message explains the purpose. We use markedForDeletion to do soft deletes and recently discovered that since preloaded apps don't have this internal field, they don't get loaded. The change above (orWhereNull...) is done for the same reason.

Reply:
Oh, I thought both commits had the same message when I skimmed the commit page, turns out they just share the same first three words :(

Reply:
lol, sorry for the confusion :)